utility.py

```diff
@@ -21,7 +22,7 @@ def exec_command(command, directory, continue_on_error=True):
         Execute command.
     """
     try:
-        check_call(command.split(), cwd=directory)
+        check_call(shlex.split(command, posix=False), cwd=directory)
```

Comment:
By leveraging `shlex` (a native Python module) we're able to process commands with quotations as well as retain the quotations using `posix=False`. This is necessary for our new call to stable unit tests that needs to add `"not unstable"` to the testing command line call.